soak

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+
+for run in {1..1000}
```

Comment:
That's a nice script! Our Makefile has [soak-eunit](https://github.com/apache/couchdb/blob/main/Makefile#L203-L205) wonder if that still works and perhaps we could have a `soak-mango` there too (but as a separate PR perhaps).

Reply:
> That's a nice script! Our Makefile has [soak-eunit](https://github.com/apache/couchdb/blob/main/Makefile#L203-L205) wonder if that still works and perhaps we could have a `soak-mango` there too (but as a separate PR perhaps).

oops that shouldn't have been added!

Reply:
Removed in https://github.com/apache/couchdb/pull/4581/commits/9538395451acf72d028c0db2e4cbc9277dcaa589